spot_driver/test/test_spot_driver.py

```diff
@@ -126,6 +136,489 @@ def test_wrapped_commands(self) -> None:
+            feedback.feedback.named_arm_position_feedback.status.STATUS_UNKNOWN
+        )
+        self.assertEqual(self.spot_ros2._process_synchronized_arm_command_feedback(feedback), GoalResponse.IN_PROGRESS)
+
+        feedback.feedback.named_arm_position_feedback.status.value = (
+            feedback.feedback.named_arm_position_feedback.status.STATUS_IN_PROGRESS
+        )
+        self.assertEqual(self.spot_ros2._process_synchronized_arm_command_feedback(feedback), GoalResponse.IN_PROGRESS)
+
+        # Should this status return a GoalResponse.FAILED?
+        feedback.feedback.named_arm_position_feedback.status.value = (
+            feedback.feedback.named_arm_position_feedback.status.STATUS_STALLED_HOLDING_ITEM
+        )
+        self.assertEqual(self.spot_ros2._process_synchronized_arm_command_feedback(feedback), GoalResponse.IN_PROGRESS)
+
+        """ Testing arm velocity feedback """
+        feedback.feedback.feedback_choice = feedback.feedback.FEEDBACK_ARM_VELOCITY_FEEDBACK_SET
+        self.assertEqual(self.spot_ros2._process_synchronized_arm_command_feedback(feedback), None)
+
+        """ Testing arm gaze feedback """
+        feedback.feedback.feedback_choice = feedback.feedback.FEEDBACK_ARM_GAZE_FEEDBACK_SET
+
+        feedback.feedback.arm_gaze_feedback.status.value = (
+            feedback.feedback.arm_gaze_feedback.status.STATUS_TRAJECTORY_COMPLETE
+        )
+        self.assertEqual(self.spot_ros2._process_synchronized_arm_command_feedback(feedback), GoalResponse.SUCCESS)
+
+        feedback.feedback.arm_gaze_feedback.status.value = feedback.feedback.arm_gaze_feedback.status.STATUS_UNKNOWN
+        self.assertEqual(self.spot_ros2._process_synchronized_arm_command_feedback(feedback), GoalResponse.IN_PROGRESS)
+
+        feedback.feedback.arm_gaze_feedback.status.value = feedback.feedback.arm_gaze_feedback.status.STATUS_IN_PROGRESS
+        self.assertEqual(self.spot_ros2._process_synchronized_arm_command_feedback(feedback), GoalResponse.IN_PROGRESS)
+
+        # Should this status return a GoalResponse.FAILED?
+        feedback.feedback.arm_gaze_feedback.status.value = (
+            feedback.feedback.arm_gaze_feedback.status.STATUS_TOOL_TRAJECTORY_STALLED
+        )
+        self.assertEqual(self.spot_ros2._process_synchronized_arm_command_feedback(feedback), GoalResponse.IN_PROGRESS)
+
+        """ Testing arm stop feedback """
+        feedback.feedback.feedback_choice = feedback.feedback.FEEDBACK_ARM_STOP_FEEDBACK_SET
+
+        self.assertEqual(self.spot_ros2._process_synchronized_arm_command_feedback(feedback), None)
+
+        """ Testing arm drag feedback """
+        feedback.feedback.feedback_choice = feedback.feedback.FEEDBACK_ARM_DRAG_FEEDBACK_SET
+
+        feedback.feedback.arm_drag_feedback.status.value = feedback.feedback.arm_drag_feedback.status.STATUS_DRAGGING
+        self.assertEqual(self.spot_ros2._process_synchronized_arm_command_feedback(feedback), GoalResponse.SUCCESS)
```

Comment:
Is this SUCCESS or IN_PROGRESS?  There doesn't seem to be any IN_PROGRESS return